news/2020-11-14-new-featured-artist-blood-code.md

```diff
@@ -14,27 +14,27 @@ We've managed to snag a collection of **12** tracks from his body of work for yo
 
 Check out a sample of what's new below:
 
-### POP SHOVE-IT FEAT. TOFUKU
+### BLOOD CODE, TOFUKU - POP SHOVE-IT
 
 Check out [this showcase set](https://osu.ppy.sh/beatmapsets/1294825) hosted by [Hobbes2](https://osu.ppy.sh/users/8157492) from the Mappers' Guild, featuring difficulties from [Gero](https://osu.ppy.sh/users/1467715)!
 
 <audio controls>
     <source src="https://assets.ppy.sh/artists/115/previews/2989.mp3" type="audio/mpeg">
 </audio>
 
-### Take Me Home (feat. Wishlyst)
+### BLOOD CODE, Wishlyst - Take Me Home (feat. Wishlyst)
```

Comment:
intended?

Reply:
this is a 1-1 match on what's in the DB after being harvested from the osz, so yes